tyk-control-plane/README.md

```diff
@@ -0,0 +1,793 @@
+
+#### Sharding
+
+Configure the gateways to load APIs with specific tags only by enabling `tyk-gateway.gateway.sharding.enabled`, and set `tags` to comma separated lists of matching tags.
+
+```yaml
+    # Sharding gateway allows you to selectively load APIs to specific gateways.
+    # If enabled make sure you have at least one gateway that is not sharded.
+    # Also be sure to match API segmentation tags with the tags selected below.
+    sharding:
+      enabled: true
+      tags: "edge,dc1,product"
+```
+
+#### Deploy additional gateway groups
+
+`tyk-control-plane` chart manages one Gateway Deployment in the same namespace as Tyk Dashboard. 
+You can flexibly deploy additional gateways using `tyk-gateway` component chart. With gateway sharding, it is useful for:
+- Deploy Gateways in different networks,
+- Deploy Gateways with different resources and autoscaling parameters,
+- Allow different teams to manage their own Gateway instances in their own namespace.
+
+Here is an example configuration for `tyk-gateway` `values.yaml`.
+```yaml
+global:
+  redis:
+    addrs:
+      - tyk-redis-master.tyk-control-plane.svc:6379       # New Gateway groups should connect to the same Redis
+    pass: "xxxxxxx"
+
+gateway:
+  # If this option is set to true, it will enable polling the Tyk Dashboard service for API definitions
+  useDashboardAppConfig:
+    enabled: true
+      # Set it to the URL to your Dashboard instance (or a load balanced instance)
+      # The URL needs to be formatted as: http://dashboard_host:port
+      # It is used to set TYK_GW_DBAPPCONFOPTIONS_CONNECTIONSTRING
+    dashboardConnectionString: "http://dashboard-svc-tyk-tyk-dashboard.tyk-control-plane.svc:3000"
+
+      # This option is required if Policy source is set to Tyk Dashboard (`service`).
+      # Set this to the URL of your Tyk Dashboard installation.
+      # The URL needs to be formatted as: http://dashboard_host:port.
+      # It is used to set TYK_GW_POLICIES_POLICYCONNECTIONSTRING
+    policyConnectionString: "http://dashboard-svc-tyk-tyk-dashboard.tyk-control-plane.svc:3000"
+
+  ...
+
+  # Sharding gateway allows you to selectively load APIs to specific gateways.
+  # If enabled make sure you have at least one gateway that is not sharded.
+  # Also be sure to match API segmentation tags with the tags selected below.
+  sharding:
+    enabled: true
+    tags: "gw-dmz"
+
+  ...
+
+  # analyticsEnabled property is used to enable/disable analytics.
+  # If set to empty or nil, analytics will be enabled/disabled based on `global.components.pump`.
+  analyticsEnabled: "true"
+
+  # used to decide whether to send the results back directly to Tyk without a hybrid pump
+  # if you want to send analytics to control plane instead of pump, change analyticsConfigType to "rpc"
+  analyticsConfigType: ""
```

Comment:
Shouldn't it be rpc by default? Or can you add TODO to update it when we work on analytics